automation_testing/pages/SaveAsComponent.ts

```diff
@@ -0,0 +1,97 @@
+import { expect, Locator, Page } from '@playwright/test';
+import { url } from '../utils/formUrls';
+
+export class SaveAsComponent {
+    readonly page: Page;
+    readonly saveAsBtn: Locator;
+    readonly saveAsOptions: Locator;
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 1 tab but found 4 spaces.](https://app.codacy.com/gh/simpleviewinc/sv-mosaic/pullRequest?prid=9498610)
```suggestion
	readonly saveAsOptions: Locator;
```